doc/examples.md

```diff
@@ -19,16 +21,42 @@ This config will write a single service unit (shown below) with the contents of
 }
 ```
 
-### example.service
+#### example.service
 
-```
+```INI
 [Service]
 Type=oneshot
 ExecStart=/usr/bin/echo Hello World
 
 [Install]
 WantedBy=multi-user.target
 ```
+### Modifying Services
+
+This config will add a [systemd unit drop-in](https://coreos.com/os/docs/latest/using-systemd-drop-in-units.html) to modify an existing service. It enables the `systemd-networkd` service and sets its environment variable `SYSTEMD_LOG_LEVEL` to `debug`.
```

Comment:
`systemd-networkd` is typically already enabled. Go ahead and drop that part. Just the dropin should be fine.